### PR TITLE
[Improvement] ec2/ec2-auto-recovery - Support At-Rest-Encryption (breaking change)

### PR DESCRIPTION
**Pull request by michaelwittig**

### CODE REVIEW

File: ec2/ec2-auto-recovery.yaml [hunk 1]
```diff
@@ -139,6 +145,13 @@ Parameters:
     Description: 'Optional comma-delimited list of IAM managed policy ARNs to attach to the instance''s IAM role'
     Type: String
     Default: ''
+  RootVolumeSize:
```
Comment (andreaswittig) [inline ec2/ec2-auto-recovery.yaml]: The new parameter is missing in the documentation?

Comment (michaelwittig) [reply ec2/ec2-auto-recovery.yaml]: we don't have a documentation per parameter in this project?

Comment (andreaswittig) [reply ec2/ec2-auto-recovery.yaml]: Sorry, mixed that up.

**Comment by michaelwittig** [comment]
template is deprecated. will not merge

**Pull request closed without merging**